tests/unit/test-cmd/test.build.js

```diff
@@ -320,6 +321,44 @@ describe('build', () => {
     }
   ));
 
+  it('raises a UsageError if zip file exists', () => {
+    return withTempDir(
+      (tmpDir) => {
+        const testFileName =
+          path.join(tmpDir.path(), 'minimal_extension-1.0.zip');
+        return fs.writeFile(testFileName, 'test')
+          .then(() =>
+        build({
+          sourceDir: fixturePath('minimal-web-ext'),
+          artifactsDir: tmpDir.path(),
+        }))
+          .catch ((error) => {
+            assert.instanceOf(error, UsageError);
+          });
+
+      });
+  });
+
+  it('overwrites zip file if it exists', () => {
+    return withTempDir(
+      (tmpDir) => {
+        const testFileName =
+          path.join(tmpDir.path(), 'minimal_extension-1.0.zip');
+        return fs.writeFile(testFileName, 'test')
+          .then(() =>
+        build({
```

Comment:
@harikishen can you indent this `build({...})` a bit to make this lines nicer to read?